<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-H4ODDOJJ" ignored="3" name="MyTest1" tests="2" failures="2" timestamp="2020-09-09T16:35:57 BST" time="0.165" errors="0">
  <testcase name="verifyTitle" time="0.022" classname="testNG.PHPTests">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at testNG.PHPTests.verifyTitle(PHPTests.java:40)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- verifyTitle -->
  <testcase name="verifyButtons" time="0.013" classname="testNG.PHPTests">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at testNG.PHPTests.verifyButtons(PHPTests.java:46)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- verifyButtons -->
  <testcase name="verifyInvalidEmailMsg" time="0.0" classname="testNG.PHPTests">
    <ignored/>
  </testcase> <!-- verifyInvalidEmailMsg -->
  <testcase name="verifyLinksAfterLogin" time="0.0" classname="testNG.PHPTests">
    <ignored/>
  </testcase> <!-- verifyLinksAfterLogin -->
  <testcase name="verifyValidEmailMsg" time="0.0" classname="testNG.PHPTests">
    <ignored/>
  </testcase> <!-- verifyValidEmailMsg -->
</testsuite> <!-- MyTest1 -->
